angular.json

```diff
@@ -25,12 +25,15 @@
             ],
             "styles": [
               "src/styles.css",
+              "node_modules/font-awesome/css/font-awesome.css",
               "node_modules/bootstrap/dist/css/bootstrap.min.css"
             ],
             "scripts": [
-              "node_modules/jquery/dist/jquery.min.js",
-              "node_modules/bootstrap/dist/js/bootstrap.min.js",
```

Comment:
Why remove bootstrap & jquery?

Reply:
Removed because we are not using these pieces of bootstrap and jquery. All we are using from bootstrap is the .css which is loaded up above.